### DISCUSSION TITLE
Remix Run + React Framework: Form submission is blank

### DISCUSSION THREAD

**Discussion by wengzilla** [remix-run/remix#5027] [Q&A] [answered]
Have been trying to test out the Remix JS framework's form submission and cannot seem to get it to work. I must be missing something obvious, but have been staring at it for hours and haven't been able to find it:

Below is a screenshot of what I'm seeing... I've got a very simple form that has one input text field called "name". When I submit it, I can see that the Form Data includes `name: test`, but then on my server logs, I see `FormData {}` when I do `console.log(body)`.

![Screen Shot 2023-01-09 at 1 48 49 AM](https://user-images.githubusercontent.com/219947/211281691-e7cedf25-a80e-44b3-8dfc-6b515fd196f4.png)

Does anyone know why there is this inconsistency?

**Comment by machour** [comment, marked as answer]
This is just because you (sadly) can't log FormData like this as it won't reveal what it's holding. 

Try console.log(formData.get("your field");

**Reply by wengzilla** [reply]
Nailed it! Thanks so much for the quick help.

I'm surprised no one else ran into this issue, but hopefully this thread helps someone else down the road. Thanks again!
